src/PostgREST/Config.hs

```diff
@@ -206,14 +206,17 @@ readPathShowHelp = customExecParser parserPrefs opts
           |## logging level, the admitted values are: crit, error, warn and info.
           |log-level = "error"
           |
-          |## rollback all transactions by default, use for test environments
-          |## disabled by default
-          |tx-rollback-all = false
-          |
-          |## allow overriding the tx-rollback-all setting for a request by
-          |## setting the Prefer: tx=[commit|rollback] header
-          |## disabled by default
-          |tx-allow-override = false
+          |## how to terminate database transactions
+          |## possible values are:
+          |## commit (default)
+          |##   transaction is always committed, this can not be overriden
+          |## commit-allow-override
+          |##   transaction is committed, but can be overriden with Prefer tx=rollback header
+          |## rollback
+          |##   transaction is always rolled back, this can not be overriden
+          |## rollback-allow-override
+          |##   transaction is rolled back, but can be overriden with Prefer tx=commit header
+          |tx-end = "commit"
```

Comment:
Nice! I like the consolidated option. Also I'm fine with saving work on the Config type :+1: .